src/parser/jobs/war/modules/SurgingTempest.tsx

```diff
@@ -41,3 +65,62 @@ export class SurgingTempest extends Analyser {
 		return (statusUptime / fightUptime) * 100
 	}
 }
+
+export class StormsEye extends CoreGauge {
+	static override handle = 'stormseye'
+
+	@dependency private suggestions!: Suggestions
+
+	private surgingTempest = this.add(new TimerGauge({
+		maximum: 60,
```

Comment:
I'd const this, even tho it's only ever changed once.

Reply:
linter refuses to allow constanting the timergauge

Reply:
You don't const the timer gauge, you const the max value and then set `maximum: MAH_CONST,`.